card/lib/generators/card/templates/create_cards_tables.erb

```diff
@@ -0,0 +1,106 @@
+      t.string  "referee_key",              :default => "", :null => false
+      t.integer "referee_id"
+      t.string  "ref_type",    :limit => 1, :default => "", :null => false
+      t.integer "present"
+    end
+
+    add_index "<%= prefix %>card_references", ["referee_id"], :name => "<%= prefix %>card_references_referee_id_index"
+    add_index "<%= prefix %>card_references", ["referee_key"], :name => "<%= prefix %>card_references_referee_key_index"
+    add_index "<%= prefix %>card_references", ["referer_id"], :name => "<%= prefix %>card_references_referer_id_index"
+
+    create_table "<%= prefix %>card_revisions", :force => true do |t|
+      t.datetime "created_at", :null => false
+      t.integer  "card_id",    :null => false
+      t.integer  "creator_id", :null => false
+      t.text     "content",    :null => false
+    end
+
+    add_index "<%= prefix %>card_revisions", ["card_id"], :name => "<%= prefix %>revisions_card_id_index"
+    add_index "<%= prefix %>card_revisions", ["creator_id"], :name => "<%= prefix %>revisions_created_by_index"
+
+    create_table "<%= prefix %>cards", :force => true do |t|
+      t.string   "name",                :null => false
+      t.string   "key",                 :null => false
+      t.string   "codename"
+      t.integer  "left_id"
+      t.integer  "right_id"
+      t.datetime "created_at",          :null => false
+      t.datetime "updated_at",          :null => false
+      t.integer  "creator_id",          :null => false
+      t.integer  "updater_id",          :null => false
+      t.string   "read_rule_class"
+      t.integer  "read_rule_id"
+      t.integer  "references_expired"
+      t.boolean  "trash",               :null => false
+      t.integer  "type_id",             :null => false
+      t.text     "db_content"
+    end
+
+    add_index "<%= prefix %>cards", ["key"], :name => "<%= prefix %>cards_key_index", :unique => true
+    add_index "<%= prefix %>cards", ["left_id"], :name => "<%= prefix %>cards_left_id_index"
+    add_index "<%= prefix %>cards", ["name"], :name => "<%= prefix %>cards_name_index"
+    add_index "<%= prefix %>cards", ["read_rule_id"], :name => "<%= prefix %>cards_read_rule_id_index"
+    add_index "<%= prefix %>cards", ["right_id"], :name => "<%= prefix %>cards_right_id_index"
+    add_index "<%= prefix %>cards", ["type_id"], :name => "<%= prefix %>cards_type_id_index"
+
+    create_table "<%= prefix %>schema_migrations_core_cards", :id => false, :force => true do |t|
+      t.string "version", :null => false
+    end
+
+    add_index "<%= prefix %>schema_migrations_core_cards", ["version"], :name => "<%= prefix %>unique_schema_migrations_cards", :unique => true
+
+    create_table "schema_migrations_deck_cards", :id => false, :force => true do |t|
+      t.string "version", :null => false
+    end
+
+    add_index "<%= prefix %>schema_migrations_deck_cards", ["version"], :name => "<%= prefix %>unique_schema_migrations_deck_cards", :unique => true
+  end
```

Comment:
To be resolved: loading seed/fixture data into these tables and running migrations from gems (particularly when updating gems to a newer version).

I did split the rake tasks so that loading the seed data is a distinct task from loading the schema, which we should prevent when we are embedded.